core/download_process.py

```diff
@@ -45,6 +45,25 @@ class DownloadProcess:
     def __init__(self, links: List[str], total_links: int, session: requests.Session,
                  http2: bool = False, max_retries: int = 5,
                  convert: bool = True, debug: bool = False):
+        """Initialize Object of DownloadProcess.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)